### DISCUSSION TITLE
Muhammad Lahia Ardhan | lahi

### DISCUSSION THREAD

**Discussion by lahiardhan** [onlyphantom/llm-python#36] [Practicum: Generative AI]
https://colab.research.google.com/drive/1T-SbyqOI-nf04HYHqVVnjpYAvTOsAYHL?usp=sharing

**Comment by onlyphantom** [comment]
Well done! Very detailed prompt template! 

Marked as passed so you should receive your certificate soon!

**Reply by lahiardhan** [reply]
Thank you so much Sam!
